website/versioned_sidebars/version-0.11-sidebars.json

```diff
@@ -99,6 +99,10 @@
         {
           "type": "doc",
           "id": "version-0.11/plugins/colorlog"
+        },
+        {
+          "type": "doc",
+          "id": "version-0.11/plugins/ax_sweeper"
```

Comment:
Sort plugins alphabetically. (At least we have to start grouping them by type).

Reply:
pending on https://github.com/facebookresearch/hydra/pull/321#discussion_r376829332